interfaces/builtin/opengl.go

```diff
@@ -74,6 +74,8 @@ unix (send, receive) type=dgram peer=(addr="@nvidia[0-9a-f]*"),
 @{PROC}/devices r,
 /sys/devices/system/memory/block_size_bytes r,
 unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
+/{dev,run}/shm/cuda.* rw,
```

Comment:
This rule is not wonderful, but we already allow something similar for pulseaudio and browser-support, so 'ok'.